torch/optim/lr_scheduler.py

```diff
@@ -487,10 +487,16 @@ def _get_closed_form_lr(self):
 
 
 class LinearLR(_LRScheduler):
-    """Decays the learning rate of each parameter group by linearly changing small
-    multiplicative factor until the number of epoch reaches a pre-defined milestone: total_iters.
-    Notice that such decay can happen simultaneously with other changes to the learning rate
-    from outside this scheduler. When last_epoch=-1, sets initial lr as lr.
+    r"""Decays the learning rate of each parameter group by linearly changing small
+    multiplicative factor until the number of epoch reaches a pre-defined milestone: total_iters,
+    or :math:`T_\text{max}`. :math:`K_\text{start}` and :math:`K_\text{end}`denote the start
```

Comment:
I think you're missing a space that breaks the rendering: https://docs-preview.pytorch.org/71231/generated/torch.optim.lr_scheduler.LinearLR.html?highlight=linearlr#torch.optim.lr_scheduler.LinearLR

You can see this rendered doc in the "  Preview docs built from this PR" link in the bot comment on the PR.

Reply:
Thanks for the heads up! Pushed the fix, and seems like things are rendering correctly now ([link](https://docs-preview.pytorch.org/71231/generated/torch.optim.lr_scheduler.LinearLR.html?highlight=linearlr#torch.optim.lr_scheduler.LinearLR)). I'm assuming the `numba` import errors are unrelated to this PR?

Reply:
The eta_0 part still does not render properly right?

Reply:
Yes CI is not in a good shape today: https://hud.pytorch.org/ci/pytorch/pytorch/master you can ignore

Reply:
> The eta_0 part still does not render properly right?

Is it not? It does look a bit awkward, but I assumed that's the way subscripts are rendered. Am I missing something here?

```
When last_epoch=-1, sets initial
     lr :math:`\eta_0` as lr.

.. math::
         \eta_t = \eta_0 \left(K_\text{start} + \frac{T_\text{curr}(K_\text{end} -
```

Thanks in advance!